src/js/expandButton.js

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2021 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
+*/
+
+"use strict";
+
+// The expand-collapse button component. This button is only available on the mobile view.
+fluid.defaults("fluid.expandButton", {
+    gradeNames: "fluid.button",
+    model: {
+        expanded: true
+    },
+    modelRelay: {
+        toggleOnActivate: {
+            source: "activate",
+            target: "expanded",
+            singleTransform: "fluid.transforms.toggle"
+        },
+        ariaExpanded: {
+            source: "expanded",
+            target: "dom.container.attrs.aria-expanded"
+        },
+        ariaLabel: {
+            source: "expanded",
+            target: "dom.container.attrs.aria-label",
+            func: expanded => expanded ? "collapse" : "expand"
```

Comment:
My understanding ([based on recent reading](https://kittygiraudel.com/2021/03/13/gorillas-nav-a-case-study/#shut-it-down)) is that because the `aria-expanded` attribute indicates state, this label should be static and should explain _what_ is being collapsed/expanded. Currently when interacting with this component, VoiceOver reads:

"expand, collapsed, button"

…and then:

"collapse, expanded, button"

Which doesn't tell me anything about what the button is actually controlling.